devicetypes/smartthings/everspring-outdoor-floodlight.src/everspring-outdoor-floodlight.groovy

```diff
@@ -0,0 +1,162 @@
+			}
+		}
+		tileAttribute("motion", "device.motion", decoration: "flat", width: 2, height: 2) {
+			attributeState("active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC")
+			attributeState("inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#CCCCCC")
+		}
+		valueTile("illuminance", "device.illuminance", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "illuminance", label: '${currentValue} lux', backgroundColors: [
+					[value: 40, color: "#999900"],
+					[value: 100, color: "#CCCC00"],
+					[value: 300, color: "#FFFF00"],
+					[value: 500, color: "#FFFF33"],
+					[value: 1000, color: "#FFFF66"],
+					[value: 2000, color: "#FFFF99"],
+					[value: 10000, color: "#FFFFCC"]
+			]
+		}
+
+		main "switch"
+		details(["switch", "motion", "illuminance"])
+	}
+}
+
+def initialize() {
+	sendEvent(name: "checkInterval", value: 2 * 4 * 60 * 60 + 24 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	sendEvent(name: "motion", value: "inactive", displayed: false)
+}
+
+def installed() {
+	initialize()
+}
+
+def updated() {
+	initialize()
+}
+
+def configure() {
+	[
+			secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
+			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x03)),
+			secure(zwave.switchBinaryV1.switchBinaryGet()),
+			secure(zwave.configurationV1.configurationSet(parameterNumber: 3, size: 2, scaledConfigurationValue: 600)) //enables illuminance report every 10 minutes
+	]
+}
+
+def parse(String description) {
+	def result = []
+	if (description.startsWith("Err")) {
+		result = createEvent(descriptionText:description, isStateChange:true)
+	} else {
+		def cmd = zwave.parse(description)
+		if (cmd) {
+			result += zwaveEvent(cmd)
+		}
+	}
+	log.debug "Parse returned: ${result}"
+	result
+}
+
+def on() {
+	[
+			secure(zwave.switchBinaryV1.switchBinarySet(switchValue: 0xFF)),
+			"delay 500",
+			secure(zwave.switchBinaryV1.switchBinaryGet())
+	]
+}
+
+def off() {
+	[
+			secure(zwave.switchBinaryV1.switchBinarySet(switchValue: 0x00)),
+			"delay 500",
+			secure(zwave.switchBinaryV1.switchBinaryGet())
+	]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand()
+	if (encapsulatedCommand) {
+		zwaveEvent(encapsulatedCommand)
+	} else {
+		log.warn "Unable to extract encapsulated cmd from $cmd"
+		createEvent(descriptionText: cmd.toString())
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd) {
+	def map = [name: "switch"]
+	map.value = cmd.value ? "on" : "off"
+	map.descriptionText = "${device.displayName} light has been turned ${map.value}"
+	createEvent(map)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
```

Comment:
Don't you have to specify the command class version of 0x71 commands in  `parse()` for this method to be called?

Reply:
Isn't Notification V3 a latest version supported by us? It would seems so, at least according to https://graph.api.smartthings.com/ide/doc/zwave-utils.html

Reply:
@PKacprowiczS is correct

Reply:
Aaah yeah, I forgot that we treat 0x71 uniquely to be mapped to V3 by default.